docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,232 @@
+
+```
+{ span.http.status_code >= 200 && span.http.status_code < 300 } | count() > 2
+```
+
+In this example, the search reduces traces to those spans where:
+
+* `http.status_code` is in the range of `200` to `299` and
+* the number of matching spans within a trace is greater than two.
+
+Queries select sets of spans and filter them through a pipeline of aggregators and conditions. If, for a given trace, this pipeline produces a spanset then it is included in the results of the query.
+
+
+## Selecting spans
+
+TraceQL differentiates between two types of span data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs. You can use intrinsics and attributes to build filters and select spans.
+
+In TraceQL, curly brackets `{}` always select a set of spans from the current trace. They are commonly paired with a condition to reduce the spans being passed in.
+
+
+### Intrinsic fields
+
+Intrinsic fields are fundamental to spans. These fields can be referenced when selecting spans. Note that custom attributes are prefixed with `.`, `span.` or `resource.` whereas intrinsics are typed directly.
+
+
+| **Operation** | **Type** | **Definition**                        | **Example**            |
+|---------------|----------|---------------------------------------|------------------------|
+| status        | string   | status values are error, ok, or unset | { status = ok }        |
+| duration      | duration | end - start time of the span          | { duration > 100ms }   |
+| name          | string   | operation or span name                | { name = "HTTP POST" } |
+
+### Attribute fields
+
+Attribute fields are derived from the span and can be customized. Process and span attribute types are [defined by the attribute itself](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/common/v1/common.proto#L30-L38), whereas intrinsic fields have a built-in type. You can refer to dynamic attributes (also known as tags) on the span or the span's resource.
+
+Attributes in a query start with a span or resource scope depending on what exactly you want to query. This provides significant performance benefits because it allows Tempo to only scan the data you are interested in. To find traces with the `GET HTTP` method, your query could look like this:
+
+```
+{ span.http.method = "GET" }
+```
+
+<p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span resources and attributes" /></p>
+
+#### Examples
+
+Find traces that passed through the `prod` namespace:
+```
+{ resource.namespace = "prod" }
+```
+
+Find any database connection string that goes to a Postgres or MySQL database:
+```
+{ span.db.system =~ "postgresql|mysql" }
+```
+
+### Unscoped attribute fields
+
+Attributes can be unscoped if you are unsure if the requested attribute exists on the span or resource. When possible, use scoped  instead of unscoped attributes.
```

Comment:
```suggestion
Attributes can be unscoped if you are unsure if the requested attribute exists on the span or resource. When possible, use scoped instead of unscoped attributes.
```